apps/remix-ide/src/app/files/fileManager.js

```diff
@@ -22,7 +22,7 @@ const profile = {
   icon: 'assets/img/fileManager.webp',
   permission: true,
   version: packageJson.version,
-  methods: ['file', 'exists', 'open', 'writeFile', 'readFile', 'copyFile', 'copyDir', 'rename', 'mkdir', 'readdir', 'remove', 'getCurrentFile', 'getFile', 'getFolder', 'setFile', 'switchFile', 'refresh'],
+  methods: ['file', 'exists', 'open', 'writeFile', 'readFile', 'copyFile', 'copyDir', 'rename', 'mkdir', 'readdir', 'remove', 'getCurrentFile', 'getFile', 'getFolder', 'setFile', 'switchFile', 'refresh', 'getProviderOf', 'getProviderByName'],
```

Comment:
`getProviderOf` and `getProviderByName` returns a complex object and won't pass through the iframe. but I guess it's fine  if we only plan to use it for internal usage.

Reply:
that is the case yes. There was no other way...

Reply:
I could also expose all the provider methods of course...